Check-in Failing for ZZZ

Check-in works fine for GI and HSR but is throwing an error for ZZZ.

```
zzz Response { data: null, message: '活动已结束 (-500012)', retcode: -500012 }

file:///home/runner/work/hoyolab-auto-daily/hoyolab-auto-daily/index.js:116
    throw new Error('Error(s) occured.')
          ^

Error: Error(s) occured.
    at main (file:///home/runner/work/hoyolab-auto-daily/hoyolab-auto-daily/index.js:116:11)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async file:///home/runner/work/hoyolab-auto-daily/hoyolab-auto-daily/index.js:183:1

Node.js v20.17.0
Error: Process completed with exit code 1.
```

--- a/index.js
+++ b/index.js
@@ -73,6 +73,8 @@ async function main() {
     headers.set('sec-fetch-site', 'same-site')
     headers.set('sec-gpc', '1')
 
+    headers.set("x-rpc-signgame": game)
+
     headers.set('user-agent', 'Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/126.0.0.0 Safari/537.36')
 
     const res = await fetch(url, { method: 'POST', headers, body })

